src/stories/Recipes/Layout/Layout.mdx

```diff
@@ -0,0 +1,75 @@
+import { Primary, ArgsTable, Canvas, Story } from "@storybook/addon-docs";
```

Comment:
Let's add a new example in here and the playground so we can show a layout with the sidebar on the right